code/modules/jobs/job_types/_job.dm

```diff
@@ -191,7 +191,12 @@
 	return
 
 /mob/living/carbon/human/on_job_equipping(datum/job/equipping)
-	var/datum/bank_account/bank_account = new(real_name, equipping, dna.species.payday_modifier)
+	var/payday
+	if(CONFIG_GET(flag/equal_nonhuman_wages) || dna.species.id == SPECIES_HUMAN)
```

Comment:
snowflaking. this should be a list or species var, not a `== species_human` snowflake check.